### PR TITLE
RGRIDT-861: Adding the ability to export with a specific file name.

### PR DESCRIPTION
**Pull request by rugoncalves**
This PR is for adding the ability for developers to provide the filename with which the excel/csv file will be exported with.

### What was happening
* The name of the excel/csv was hardcoded in the code

### What was done
* A new input parameter was added to the feature export, name "Filename", with default value "DataGridReactive".
* A new function was created to handle the name properly

### Test Steps
1. Add a filename in the ContextMenu option "Export to Excel"
2. Validate that the file exported has the filename + extension passed in OutSystems
3. Leave the filename field empty
4. Validate that the output file has the filename 'DataGridReactive' + extension.

### Checklist
* [x] tested locally
* [x] documented the code
* [x] clean all warnings and errors of eslint
* [x] requires changes in OutSystems (if so, provide a module with changes)
* [x] requires new sample page in OutSystems (if so, provide a module with changes)

### CODE REVIEW

File: code/src/WijmoProvider/Features/Export.ts [hunk 1]
```diff
@@ -22,6 +23,14 @@ namespace WijmoProvider.Feature {
             );
         }
 
+        private _handleFilename(fileName: string, isCSV = false): string {
+            if (fileName === undefined || fileName === '') {
```
Comment (glundgren93) [inline code/src/WijmoProvider/Features/Export.ts:27]: Wouldn't be better to set the value for fileName as a default value on the parameter itself?

fileName: string = 'DataGridReactive'

Comment (rugoncalves) [reply code/src/WijmoProvider/Features/Export.ts:27]: That only works for when the filename is _undefined_. If the name is '' (empty), then the default value would not be used...

**Review by tmbp95 — APPROVED** [review]

**Review by glundgren93 — APPROVED** [review]

**Comment by zipzapduo** [comment]
Is it ready for download?

**Comment by glundgren93** [comment]
> Is it ready for download? I mean is it already in Outsystems forge?

Not yet. We plan on making a release next week.

**Pull request merged**